workloads/pod_client.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
```

Comment:
should we create a folder for windowns or no need ?

Reply:
no need atm and I was leaning towards even moving out the amd64 folder as lot of workloads are already copied in to arm64